### ISSUE TITLE
Develop workflow for common meta DB tables

### ISSUE THREAD

**Issue by nesnoj** [rl-institut/WAM#94]
The following meta app tables are used by different apps:
- meta_assumption
- meta_category
- meta_source

When updating the WAMdev/prod, the only thing that doesn't get updated are the DB tables. While app tables are updated by the responsible app admin, common tables are not.
Over the time, the content of those tables diverged which is an annoying problem. We have to make sure the content always matches and there's a workflow you can rely upon. Therefore, we need a

**Code of conduct (CoC)**
1. Modifications of meta tables must be done first on WAMdev
2. Use the admin interface to add new datasets to meta tables as custom scripts or restore actions are likely to destroy or overwrite datasets used by other apps.
3. Check whether WAMprod's meta tables contain datasets which are not included in WAMdev's tables and notify the blameworthy app admin to strictly follow this CoC ;).
4. Update WAMprod meta tables whenever necessary either manually (e.g. admin interface) or using script `<TBD>`.

Awaiting your feedback.

Related meta app issues: #86 #22 #16 #7

**Comment by nesnoj** [comment]
@henhuy, if you don't mind I'll update the sources and category tables on WAMprod manually by tomorrow as I need them for StEmp ABW v1.0

**Comment by henhuy** [comment]
What do you mean by manually?
Changing ids in table?
For me it's okay. Didn't have time for #86 yet...

**Comment by nesnoj** [comment]
> What do you mean by manually?
> Changing ids in table?

Yepp, our sources are stored on WAMdev, I'll manually add them to WAMprod.
Can I kick all of your sources on WAMdev? They seem outdated to me and you have the latest list on WAM, right?

> For me it's okay. Didn't have time for #86 yet...

Nevermind, just keep in mind that StEmp ABW must be modified too if you implement #86..

**Comment by henhuy** [comment]
> Can I kick all of your sources on WAMdev? They seem outdated to me and you have the latest list on WAM, right?

No problem.

> Nevermind, just keep in mind that StEmp ABW must be modified too if you implement #86..

For sure

**Comment by nesnoj** [comment]
@henhuy There're 2 categories "Warmwasser" on WAMprod, I'll delete the one which is not referenced by your sources or assumptions..

**Comment by henhuy** [comment]
remove it. I'm going to reset my assumptions and sources anyway...

**Comment by nesnoj** [comment]
I updated the meta tables on WAMdev+WAMprod, they are now in sync!
Please use the COC above for future work.
I backupped the current state on the WAM server to `/backup/WAM_meta_tables_190927/`
